liota/dccs/iotcc.py

```diff
@@ -239,13 +246,15 @@ def _properties(self, msg_id, entity_type, entity_id, entity_name, timestamp, pr
             msg["body"]["property_data"].append({"propertyKey": key, "propertyValue": value})
         return msg
 
-    def _get_properties(self, msg_id, res_uuid):
+    def _get_properties(self, msg_id, ref_entity):
         return {
             "transactionID": msg_id,
             "version": self._version,
             "type": "get_properties_request",
             "body": {
-                "uuid": res_uuid
+                "kind": ref_entity.entity_type,
```

Comment:
Please provide the reason for _get_properties requiring "kind" and "name"?

Reply:
As described in the other CL, all the calls have now been standardized to use the same format (uuid, name, kind)